partners/scientific-communities.md

```diff
@@ -52,12 +61,15 @@ Our vibrant community is filled with experts in the Python packaging space. A
 partnership with us means that you will always have support if you run into
 Python packaging issues.
 
-### Future work
+:::{admonition} What's in the works
+:class: note
+
+We are currently enhancing our peer review process and working together on:
 
-We are currently working on several items to improve our review process and followup activities with package maintainers including:.
+-Adding an editorial bot to our peer review infrastructure to further streamline our processes.
+- Collecting metrics around package maintenance to track health and to help identify when packages become unmaintained.
```

Comment:
```suggestion
- Collecting metrics around package maintenance to track package health and to help identify when packages become unmaintained.
```

Reply:
both of these processes are important, but they sound kind of...sterile? slightly negative? in other words, if I'm interested in collaborating with pyOS, these make it sound like:
- pyOS doesn't have a fully worked-out process
- packages are unhealthy

the first one is fine, but I think rewording the second to be more positive, along the lines of "to track package health and ensure all pyOS packages are well-maintained"

Reply:
I'm also wondering if there's anything else we could talk about here that would be a direct benefit to potential partners - something they'd see and go "oh, that's interesting!" or "we could really use that!"

Reply:
A bit of context:

Every partner community i've talked to has a pain point of tracking these things! that is what they want. currently many have these manual badges that quickly get out of date. so us automating it and working together as a community is that "we could really use that" feature that everyone wants! the bot is more for us / review to ensure consistency. it's so easy to forget steps such as adding labels, etc.  

however - i agree that it could read as  unhealthy AND we don't have a process yet - part of the collab will be figuring out what these checks look like. 

We already did work together on our archiving policy in the peer review guide with a few communities. So the health metrics will be similar i think and will be a learning process! 

with that said i've rewritten as follows to try to address that "it sounds negative" vibe it was giving

> 
> pyOpenSci is enhancing our peer review process and working on:
> 
> - Adding an editorial bot to our peer review infrastructure to streamline our review processes.
> - Collecting package metrics to track health and also understand maintenance status.

we could consider using a different word than health. i'm just not sure what word to insert!

Reply:
@lwasser I don't think there's an issue with the word health, but rather "help identify when packages become unmaintained" - so what I'm thinking is more "help identify when packages are in need of maintenance" or something to indicate that we're keeping an eye on the issue. and if we offer assistance on intervention (finding new maintainers, etc.) mentioning that as well!